docs/query-language.md

```diff
@@ -420,3 +420,83 @@ Range(frame="stargazer", stargazer_id=1, start="2010-01-01T00:00", end="2017-03-
 Returns `{{"attrs":{},"bits":[10]}`
 
 * bits are repositories which were starred by user 1 from 2010-01-01 to 2017-03-02
+
+
+#### Range (BSI)
+
+**Spec:**
+
+```
+Range(<frame=STRING>, <FIELD_NAME, COMPARISON_OPERATOR, integer> )
+```
+
+**Description:**
+
+The to `Range` query is overloaded to also work on `field` values.
```

Comment:
The `Range` query is overloaded to work on `field` values as well as `timestamp` values.